modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3833,8 +3833,11 @@ Ref<GDScriptParserRef> GDScriptAnalyzer::ensure_cached_external_parser_for_class
 	}
 
 	String script_path = p_class->get_datatype().script_path;
+	Ref<GDScriptParserRef> parser_ref = parser->get_depended_parser_for(script_path);
+	if (!parser_ref.is_null()) {
```

Comment:
```suggestion
	if (parser_ref.is_valid()) {
```

Reply:
Thanks! Noticed that too right after I opened the PR but you were once again faster than light 😄 Fixed now.